master-userdata.tpl

```diff
@@ -2,22 +2,27 @@
 set -euo pipefail
 
 # Install Updates and Dependencies from Amazon repositories
-amazon-linux-extras install docker
-yum update -y
-yum -y install java-1.8.0-openjdk make git python3 jq
+sudo amazon-linux-extras install docker
```

Comment:
>`sudo`

Userdata runs as root, why invoke `sudo`?

Reply:
We don't need `sudo` indeed. During the tests, I found some permissions errors and while I was debugging I set `sudo` to everything. Removed and explaining the permissions problems below.